### ISSUE TITLE
[Feature Request] export playlists automatically

### ISSUE THREAD

**Issue by NatureMuncher** [ampache/ampache#3400]
## Describe what you don't like about Ampache, or what it is missing
Ampache playlists are great and very useful in Ampache. However I also happen to stream using MyMedia with Alexa, which uses the same folder of songs. The playlist export feature works well on the command-line, so what I did was to set up a cronjob every 15 minutes to export my playlists:
~~~
*/15 * * * * /var/www/html/ampache/bin/cli export:playlist -v ~/Music/playlists/ m3u > /dev/null 2>&1
~~~
Then I tell MyMedia to add the ~/Music folder to its scan. This works well.
## Describe how you would like the feature to work**
It might be useful to offer this through the GUI under:
#preferences.php?tab=playlist
such as "Store playlists on filesystem" with a field to enter your path.

**Comment by lachlan-00** [comment]
export on change?

**Comment by NatureMuncher** [comment]
Yes, exactly. Also are apostrophes supposed to be escaped or is that more of a per operating system quirk?

**Comment by lachlan-00** [comment]
It may depend on the player/file system. i'm not really sure as i only use playlist files for icecast but if they are it shouldn't take much effort to add.

apparently from a quick google we should be renaming those files [https://forum.minimserver.com/showthread.php?tid=5280]
![image](https://user-images.githubusercontent.com/1305249/205162838-6628942c-6daf-45a9-9b89-9a9c56be1eec.png)

**Comment by lachlan-00** [comment]
Database is set to utf8 by default so i think we should probably default to that for file exports

**Comment by NatureMuncher** [comment]
Oh that's interesting. That looks like it should work fine since MyMedia handles those too:

> File based playlists (M3U, M3U8, PLS, WPL) are automatically indexed if My Media finds them during a scan of a Watch Folder.
